### DISCUSSION TITLE
Q-Table custom-top issue occurs when there are three tabs

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#14527] [General - Components / Directives / etc] [answered]
Dear Quasar team,

I did a demo to illustrate this issue, could you please have a look?
https://codepen.io/Stanley-549393092/pen/XWqqWvd?editors=1010

There are three tabs and each tab contains a table. In tab1, the table contains button A. In tab2, the table contains button B.
In  tab3, the table contains button C.
Now, I click tab1, it show button A, then click tab2, it show button B, finally click tab3, it still show button B which is incorrect.
Then I change the order, click tab1, it show button A, then click tab3, it show button C, finally click tab2, it still show button C which is incorrect.
It seeas the issue only occurs when there are at lease three tabs/tables.

Regards

**Comment by pdanpdan** [comment, marked as answer]
Add different `key="..."` on each of the tables - else vue will see that all components are the same and it looks like it's not detecting that the slot is different

**Reply by 549393092** [reply]
Yeah, it works now. Thanks a lot!
